Misc/NEWS.d/next/Library/2018-10-31-15-39-17.bpo-35121.EgHv9k.rst

```diff
@@ -0,0 +1,2 @@
+Prefix domain with dot for proper subdomain validation in
```

Comment:
This describes what the code does, which is an implementation detail. A news entry should describe the change in the user visible behavior.

Reply:
Added a note about it but this is also affects when `domain_return_ok` is present. I couldn't come up with a better wording since I am not a native speaker. Suggestions welcome. This started as a bug fix but since this
turned out to be a security issue is it okay to move this to security section?

```rst
Don't send cookies of domain A without Domain attribute to domain B
when domain A is a suffix match of domain B while using a cookiejar
with :meth:`http.cookiejar.DefaultCookiePolicy` policy. Patch by
Karthikeyan Singaravelan.
```